lib/caching-precompiler.js

```diff
@@ -1,142 +1,180 @@
 'use strict';
-var fs = require('fs');
 var path = require('path');
+var fs = require('fs');
+var generateMapFileComment = require('convert-source-map').generateMapFileComment;
+var convertSourceMap = require('convert-source-map');
```

Comment:
Ah cool, that's the strategy I've been employing myself. I got thrown off here by the reordering of `fs` and `path`, looks like that plugin doesn't enforce any specific sorting within the categories?